linkerd-await.yaml

```
@@ -25,8 +25,8 @@ pipeline:
   - runs: |
       cargo fetch
       cargo auditable build --frozen --release
-      mkdir -p ${{targets.destdir}}/usr/bin
-      mv target/release/linkerd-await ${{targets.destdir}}/usr/bin
+      mkdir -p ${{targets.destdir}}/usr/lib/linkerd
+      mv target/release/linkerd-await ${{targets.destdir}}/usr/lib/linkerd
```

these should be a sub-package compats 
@xnox 
```suggestion
      mkdir -p ${{targets.destdir}}/usr/lib/linkerd
      mv target/release/linkerd-await ${{targets.destdir}}/usr/lib/linkerd
```

---

@xnox, FYI - can you ack / approve here?

---

`/usr/lib/linkerd` is valid without need for introducing a compat package (see `systemd`, `cups`, `firewalld` and others). I'd prefer upstream had used `libexec`, but it's still an acceptable path